samza-core/src/main/java/org/apache/samza/processor/SamzaContainerListener.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.processor;
+
+/**
+ * Listener interface that can be registered with the {@link org.apache.samza.container.SamzaContainer} instance in
+ * order to receive notifications.
+ */
+public interface SamzaContainerListener {
+
+  /**
+   * <p>
```

Comment:
It is so odd that <p> is mostly used for breaklines. I removed from this class. Will adjust other docs as and when I find them.